### PR TITLE
Set Adam's learning rate to 1e-3 (as in optim/adam.lua)

### PR DESCRIPTION
**Pull request by szagoruyko**
https://github.com/torch/optim/blob/master/adam.lua#L28

### CODE REVIEW

**Comment by jekbradbury** [comment]
Yeah, this has already tripped us up a couple times (e.g., https://pytorch.slack.com/archives/random/p1482451896000037). 1e-3 is the default for pretty much every other framework, I think.

**Comment by soumith** [comment]
cc: @yuandong-tian
